Client/mods/deathmatch/logic/CEvents.cpp

```diff
@@ -28,16 +28,27 @@ bool CEvents::AddEvent(const char* szName, const char* szArguments, CLuaMain* pL
     assert(szName);
     assert(szArguments);
 
-    // If it already exists, return
-    if (Get(szName))
-        return false;
+    // Get the event if it already exists
+    SEvent* pEvent = Get(szName);
 
-    // Create and add the event
-    SEvent* pEvent = new SEvent;
-    pEvent->strName = szName;
-    pEvent->strArguments = szArguments;
-    pEvent->pLuaMain = pLuaMain;
-    pEvent->bAllowRemoteTrigger = bAllowRemoteTrigger;
+    if (pEvent)
+    {
+        // If bAllowRemoteTrigger has been altered, return
+        if (pEvent->bAllowRemoteTrigger != bAllowRemoteTrigger)
+            return false;
```

Comment:
This is going to cause backwards compatibility issues with some scripts.

Reply:
Could you elaborate as to why? Current functionality will also return false when calling addEvent() multiple times for the same event. I could be missing something here of course, just curious. ^^

Reply:
Hm.. Can't remember actually. Probably some brainfart.